frontend/catalyst/device/qjit_device.py

```diff
@@ -390,9 +395,19 @@ def preprocess(
         # TOML files
         _, config = self.original_device.preprocess(execution_config)
         program = TransformProgram()
+        max_expansion = (
+            self.original_device.max_expansion
+            if hasattr(self.original_device, "max_expansion")
+            else None
+        )
```

Comment:
Similarly, no `original_device` from the new device API is going to have a `max_expansion` attribute.

Reply:
See my comment regarding tests